lxd/auth/driver_rbac.go

```diff
@@ -161,6 +163,13 @@ func (r *rbac) validateConfig() error {
 
 	r.agentUsername = val.(string)
 
+	val, ok = r.config["rbac.api.url"]
+	if !ok {
+		return fmt.Errorf("Missing RBAC API URL")
```

Comment:
Please can you use the config key name in the error yo show the user what needs fixing